spring-boot/starter/pom.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.togglz</groupId>
+    <artifactId>togglz-spring-boot</artifactId>
+    <version>2.6.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>togglz-spring-boot-starter</artifactId>
+  <name>Togglz - Spring Boot 2 Starter</name>
+  <description>Togglz - Spring Boot 2 Starter</description>
```

Comment:
Starter Dependency for Togglz in Spring Boot environment.

Reply:
Same as above.